elide-async/src/main/java/com/yahoo/elide/async/operation/AsyncQueryCallableOperation.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.async.operation;
+
+import com.yahoo.elide.ElideResponse;
+import com.yahoo.elide.async.models.AsyncAPI;
+import com.yahoo.elide.async.models.AsyncAPIResult;
+import com.yahoo.elide.async.models.AsyncQuery;
+import com.yahoo.elide.async.models.AsyncQueryResult;
+import com.yahoo.elide.async.service.AsyncExecutorService;
+import com.yahoo.elide.core.RequestScope;
+import com.yahoo.elide.core.security.User;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.net.URISyntaxException;
+import java.util.Date;
+import java.util.concurrent.Callable;
+
+/**
+ * AsyncQuery Execute Operation Interface.
+ */
+@Slf4j
+public abstract class AsyncQueryCallableOperation implements Callable<AsyncAPIResult> {
+    @Getter private AsyncExecutorService service;
+    private AsyncQuery queryObj;
+    private RequestScope scope;
+
+    public AsyncQueryCallableOperation(AsyncExecutorService service, AsyncAPI queryObj, RequestScope scope) {
+        this.service = service;
+        this.queryObj = (AsyncQuery) queryObj;
+        this.scope = scope;
+    }
+
+    @Override
+    public AsyncAPIResult call() throws URISyntaxException {
+        ElideResponse response = null;
+        log.debug("AsyncQuery Object from request: {}", this);
```

Comment:
I don't see toString implemented on this class.  I assume this will just log the object reference.